src/YoutubeAtomSticky.tsx

```diff
@@ -80,17 +80,18 @@ const stickyStyles = css`
 `;
 
 const stickyContainerStyles = (isMainMedia?: boolean) => {
-    const height = 192;
+    const videoHeight = 192;
```

Comment:
Nit but should we just inline these values now they're only used once?

Reply:
Yeah sounds good!